routing.md

```diff
@@ -305,12 +321,51 @@ DIDComm DID Document endpoints have the following format:
 
 **type**: MUST be `DIDComm`. 
 
-**serviceEndpoint**: MUST contain a URI for a transport specified in the [transports] section of this spec.
+**serviceEndpoint**: MUST contain a URI for a transport specified in the [transports] section of this spec, or a URI from Alternative Endpoints. Endpoints from the [transports] section SHOULD be used only for the reception of DIDComm messages.
 
 **routingKeys**: An optional ordered array of strings referencing keys to be used when preparing the message for transmission as specified in the [Routing] section of this spec. 
 
 #### Multiple Endpoints
 
 A DID Document may contain multiple service entries of type `DIDComm`. Unless otherwise specified, messages should be sent to endpoints in the order they are specified.
```

Comment:
I don't think this "in the order specified" is quite true. The first criterion ought to be a compatible transport for the circumstances; order of endpoints in the doc would be secondary. Suppose I have 5 DIDComm endpoints: 1= http, 2=http, 3=chat, 4=smtp, 5=bluetooth. If I'm disconnected from the internet, only bluetooth is useful, so I pick that. If I'm connected, I might pick any of endpoints 1-4, depending on whether I'm streaming large chunks of data (http), using the BasicMessage protocol (chat or smtp), etc. If I know I'm using http, I should prefer 1 over 2.

Reply:
I've updated the language to address your points.